app/api/ApiServer.js

```diff
@@ -0,0 +1,27 @@
+//This creates new axios instance that is only used by router for server-side api calls.
+//If user is authenticated we create an interceptor for all requests made from server to server that were instigated by client
+//It's basically only used for initial server side rendering
+import axios from 'axios';
+
+let clientConfig = {
+  host: process.env.HOSTNAME || '0.0.0.0',
+  port: process.env.PORT || '8080'
+};
```

Comment:
The reason I had to separate these out is because of how axios interceptors work. 

When you add an interceptor on axios instance it will get used on *all* of it's requests.

So it means I need to have two axios instances:

ApiServer one that gets initialized before configureStore in render function inside [server.js](https://github.com/Epochi/react-webpack-node/blob/master/app/server.jsx), and I put the interceptor on this axios instance, but client cannot interact with it. 

ApiClient instance gets created before configureStore inside [client.js](https://github.com/Epochi/react-webpack-node/blob/master/app/client.jsx). and works as expected. 

ApiServer lives only in server and gets used only for server to server.
ApiClient lives only in client and gets used only for client to server


I separated these out essentially because I'm scared that someone could somehow modify axios interceptors if they were created client side. But I don't know if this is possible.